### PR TITLE
Feature/pass api key to swagger - closes #76

### LINKED ISSUE #76: Create API testing functionality in documentation browser
We have integrated the Swagger UI, which includes sandbox functionality. The functionality should be improved in order to use the actual API key and the API Umbrella proxy.

The Swagger UI is embedded in the `/documentation` template and has two fields at the top:
swaggerui-urlwithapi_key

In the Swagger UI, we want to populate the `api_key` field with the current user's API key.

# User Story
```
 As Minh (API consumer), I want to test API calls in a sandbox.
```

# Definition of done

* [x] Determine how to modify the [Swagger UI](http://swagger.io/swagger-ui/) API testing functionality.
  * [x] Modifications to Swagger UI may be unnecessary, if we can pass arguments in to the iframe
* [x] Pass the APInf user API key to Swagger UI sandbox. (e.g. via an iframe argument or some other method)
* [ ] If possible, ensure the Swagger UI uses API Umbrella proxy, e.g. umbrella.apinf.io, for requests.
  * The API Umbrella proxy URL can be found in [config.json](https://github.com/apinf/api-umbrella-dashboard#settingsjson) (as `api_umbrella.base_url`)

### PR DESCRIPTION
**Pull request by frenchbread**

### CODE REVIEW

**Comment by frenchbread** [comment]
#76 related issue. @apinf/developers Please review

File: client/views/documentation/view/documentation.js [hunk 1]
```diff
@@ -29,6 +29,17 @@ Template.documentation.events({
 
     // Set Swagger document URL to selected value
     instance.swaggerDocumentUrl.set(event.target.value);
+  },
+  'load #documentation': function () {
```
Comment (mauriciovieira) [inline client/views/documentation/view/documentation.js]: We should try to adhere to some conventions, like use ' or " to name the events.
As the other events and helpers are using ", I suggest replacing ' for ".

File: client/views/documentation/view/documentation.js [hunk 2]
```diff
@@ -29,6 +29,17 @@ Template.documentation.events({
...
+
+    // gets current users object
+    var user = Meteor.user();
+
+    // gets current user's apiKey
+    var apiKey = user.profile.apiKey;
+
+    // passes api key to iFrame once it is loaded
+    $('#documentation').contents().find('#input_apiKey').val(apiKey);
```
Comment (mauriciovieira) [inline client/views/documentation/view/documentation.js]: We should try to adhere to some conventions, like use ' or " to findthe selectors.
As the other selectors are using ", I suggest replacing ' for ".

 Compare with https://github.com/apinf/api-umbrella-dashboard/blob/develop/client/views/documentation/view/documentation.js#L18

**Comment by mauriciovieira** [comment]
After these tiny changes, :+1:, feel free to merge.

**Comment by frenchbread** [comment]
@mauriciovieira Thanks!

**Comment by brylie** [comment]
@mauriciovieira is there any way I can help here?

**Comment by mauriciovieira** [comment]
@brylie I think it is done

**Comment by frenchbread** [comment]
@brylie Actually I came up with better solution for this issue after stated working on #517. 

In `public/swagger/index.html` there are commented out lines:
```
// if you have an apiKey you would like to pre-populate on the page for demonstration purposes...
 /*
    var apiKey = "myApiKeyXXXX123456789";
    $('#input_apiKey').val(apiKey);
*/
```
This would be the best place to preload user's api key. The thing is that Swagger UI is currently loaded using iframe in `client/views/documentation/view/documentation.html`
```
<iframe src="/swagger/index.html?url={{ swaggerDocumentUrl }}" width="100%" id="documentation"></iframe>
```
Meaning that file `index.html` which I mentioned above is not used at all, so approach that I am suggesting is not suitable.

In addition, it would be also possible to change `path` (e.g. `url`) to swagger.json file (line 30 in `public/swagger/index.html`)

**Pull request merged**